notebooks/ImvizExample.ipynb

```diff
@@ -2,15 +2,13 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "handed-person",
```

Comment:
I have no idea how these got into the original notebook from Tom R. But when I update it, these are removed by `jupyter`. 🤷

Reply:
Maybe a jupyter version difference? Fine to remove though :)